packages/ui/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'jsdom',
-  modulePathIgnorePatterns: ['<rootDir>/dist/'],
+  // modulePathIgnorePatterns: ['<rootDir>/dist/'],
```

Comment:
This looks like an unrelated change. Is this necessary?

Reply:
Yes, I undo that, I am trying something forgot to undo this.